### DISCUSSION TITLE
Requesting Information on the Availability of Commands for Extracting Protein Embeddings Using the ESM3 Model

### DISCUSSION THREAD

**Discussion by faezesarlakifar** [evolutionaryscale/esm#45] [Q&A]
Is there any super helpful command for ESM3, like the one for ESM-v2, which directly extracts embeddings from protein primary sequences in FASTA format?

esm/scripts/extract.py esm2_t33_650M_UR50D $input_file \
embeddings/algpred2_train_positive --repr_layers 0 32 33 --include mean per_tok

**Comment by jzrepus** [comment]
https://github.com/evolutionaryscale/esm/issues/28
